lib/collections/transform/cartOrder.js

```diff
@@ -166,5 +175,30 @@ export const cartOrderTransform = {
       }
       return itemsByShop;
     }, {});
+  },
+  cartCount() {
+    Logger.warn("this method has been deprecated and is now replaced by the getCount method");
+    return this.getCount();
+  },
+  cartShipping() {
+    Logger.warn("this method has been deprecated and is now replaced by the getShippingTotal method");
```

Comment:
Why not say "cartShipping()" has been replaced instead of "this method" so we don't have to guess which method is now getShippingTotal.

Same comment applies to all.